README.md

```diff
@@ -104,7 +104,7 @@ Example command:
 
 ### Setup
 - Clone the repository to desired directory on local computer
-> `git clone https://github.com/wacoo/DEF-CON_Summit.git`
+> `git clone https://github.com/wacoo/Awesome-Books.git`
 
 ## 👥 Author <a name="authors"></a>
```

Comment:
- [ ] Since there are multiple authors with separate commits for this project, kindly update your readme to reflect the participants and authors on this project before merging 👍